cdc/processor/processor.go

```diff
@@ -135,17 +137,17 @@ func (p *processor) AddTable(
 			zap.String("captureID", p.captureInfo.ID),
 			zap.String("namespace", p.changefeedID.Namespace),
 			zap.String("changefeed", p.changefeedID.ID),
-			zap.Int64("tableID", tableID),
+			zap.Int64("tableID", span.TableID),
```

Comment:
It looks a lot of modification about replace the `tableID` to `span.TableID`.

How about add a local variable `tableID := span.TableID` first to reduce modifications ?

Reply:
The log field will be replaced `zap.Stringer("span", span)` in later PR. I think it's okay for now, and ease my work in later PR (just search and replace).